tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -475,6 +475,8 @@ prepare_overlay()
 	# Kata systemd unit file
 	mkdir -p ./etc/systemd/system/basic.target.wants/
 	ln -sf /usr/lib/systemd/system/kata-containers.target ./etc/systemd/system/basic.target.wants/kata-containers.target
+	mkdir -p ./etc/systemd/system/kata-containers.target.wants/
+	ln -sf /usr/lib/systemd/system/dbus.socket  ./etc/systemd/system/kata-containers.target.wants/dbus.socket
```

Comment:
I know this was done that way before, but why is there an absolute path for /usr but a relative one for ./etc? I assume that it's because it's a symbolic link and you want the path as it exists on the target?

Reply:
> I know this was done that way before, but why is there an absolute path for /usr but a relative one for ./etc? I assume that it's because it's a symbolic link and you want the path as it exists on the target?

The first argument to `ln -s` is basically just a string that will be parsed at runtime when the symbolic link is traversed. The second argument is the path of the actual symbolic link we're creating. So yes, your assumption is correct.

Reply:
@gkurz Why not to insert the same logic here: https://github.com/gkurz/kata-containers/blob/eb1762e813c52d5b593bacb04932d6fbf0ca3a19/tools/osbuilder/rootfs-builder/rootfs.sh#L630?

Reply:
> @gkurz Why not to insert the same logic here: https://github.com/gkurz/kata-containers/blob/eb1762e813c52d5b593bacb04932d6fbf0ca3a19/tools/osbuilder/rootfs-builder/rootfs.sh#L630?

Hi @Vlad1mir-D ! Thanks for reporting this. I simply missed it :disappointed: . Let's address this with #6677 .